wrappers/rust/icicle-fields/icicle-goldilocks/src/lib.rs

```diff
@@ -0,0 +1,10 @@
+pub mod field;
+pub mod ntt;
+pub mod polynomials;
+// TODO: uncomment once implemented for goldilocks
+// pub mod poseidon;
+// pub mod poseidon2;
```

Comment:
Danny added poseidon2 constants in #840, who is adding goldilocks poseidon2 for the backends?